Kiosk/Bid Fulfillment/ManualCreditCardInputViewController.swift

```diff
@@ -77,14 +77,19 @@ class ManualCreditCardInputViewController: UIViewController, RegistrationSubCont
         action
             .errors // Based on errors
             .take(1) // On the first error, then forever
+            .logNext()
```

Comment:
I just thought this would be helpful (Auctions Ops/CRT can see the device logs at events).